src/pal/pal_posix.h

```diff
@@ -19,6 +19,10 @@
 #  include <unistd.h>
 #endif
 
+#ifndef LIBC_HAS_GETENTROPY
+#  include <cassert>
+#endif
```

Comment:
We aren't actually checking a dynamic condition here, so this should not be necessary.